index.html

```diff
@@ -43,22 +48,33 @@ <h1>Pick a theme</h1>
         localStorage.setItem('selected-theme', theme);
       }
     }
+    
+    const setDefaultTheme = () => {
+      themes.default = (prefersColorMediaQuery.matches === true) ? themes.dark : themes.light
```

Comment:
On line 31 the default theme is set to orange when `prefers-color-scheme` isn't supported:
```
default: 'orange' // Theme to load when prefers-colors-scheme isn’t supported
```
Would this line overwrite it to the light theme (pink) when the media query isn't supported?

Reply:
Oh good question, should add that to the todo list:

- [ ] check which theme loads if pcs isn’t supported

Reply:
for what it’s worth, if you check your website in IE11 none of the themes load at the moment.

Reply:
ok, according to my [test page](https://codepen.io/Moiety/pen/qEWXKGj/f9906c76b1b681be53025d94bb444c84) it just returns a true or false, so no errors.
<img width="575" alt="IE11 showing the output is false" src="https://github.com/user-attachments/assets/0f22204e-6a23-4eba-b6c4-e6d84f58d76e" />

Reply:
i think you’re right that the line always overwrites the thing, which is a problem. i’ll work on that.

background info:`window.matchMedia("bleh")` would return:

```
MediaQueryList {
  media: 'bleh',
  matches: false,
  onchange: null
}
```

so given that IE11 supports matchMedia it would just always return false

Reply:
i’ve pushed a fix for this. it’ll now correctly set the default theme if pcs isn’t supported.